crates/arroyo-connectors/src/mqtt/mod.rs

```diff
@@ -74,12 +74,18 @@ impl MqttConnector {
             None
         };
 
+        let max_packet_size = options
+            .pull_opt_str("maxPacketSize")?
```

Comment:
nit: for SQL options, we use underscores (so `max_packet_size`). Also, since this is a number you can pull it directly as one with `pull_opt_u64` which will allow users to not need to quote the value, and will handle errors nicely